msgcntr/messageforums-api/src/bundle/org/sakaiproject/api/app/messagecenter/bundle/Messages.properties

```
@@ -685,6 +688,10 @@ pvt_replied=Already replied
 
 pvt_no_subject=No subject
 
+#SAK-46974:
+pvt_scheduler_send= Program message sending
+pvt_scheduler_send_as_email= Program the message to be sent at the selected date and time
```

How about something like:

```suggestion
pvt_scheduler_send=Schedule when this message will be sent
pvt_scheduler_send_as_email=Schedule the message to be sent at the selected date and time
```

---

@victorGomollon change this pls